### ISSUE TITLE
My communities is no longer on beta site

### ISSUE THREAD

**Issue by themarkymark-steem** [steemit/condenser#3689]
My communities used to be on the left side sidebar, and trending communities in the right.  Now trending communities are on both sidebars.

# screenshot
https://i.imgur.com/HusyKPP.png

**Comment by roadscape** [comment]
I believe this will be addressed by https://github.com/steemit/condenser/pull/3684

**Comment by gl2748** [comment]
This is actually being caused by what looks like a regression
```
Error Fetching Account Subscriptions:  RPCError: Invalid Request
    at new RPCError (webpack-internal:///529:43:106)
    at eval (webpack-internal:///529:93:13)
```
Addressed in #3684

**Comment by roadscape** [comment]
https://github.com/steemit/condenser/pull/3684 merged
